Src/Eiffel/eiffel/byte_code/assign_bl.e

```diff
@@ -173,6 +173,18 @@ feature
 			expr_b: EXPR_B
 			saved_context: like context
 		do
+				-- If the source is a creation_exp_b and it's a once creation procedure
```

Comment:
If code for creation instruction should be different, it should be done when it's generated, not by modifying the byte nodes on the fly.